packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -323,10 +325,11 @@ class LabeledGlobalKey<T extends State<StatefulWidget>> extends GlobalKey<T> {
   /// Creates a global key with a debugging label.
   ///
   /// The label does not affect the key's identity.
+  // ignore: prefer_const_constructors_in_immutables , never use const for this class
   LabeledGlobalKey(this._debugLabel) : super.constructor();
 
   // Used for forwarding the constructor from GlobalKey.
-  LabeledGlobalKey._({ String debugLabel }) : _debugLabel = debugLabel, super.constructor();
+  LabeledGlobalKey._({ String debugLabel }) : this(debugLabel);
```

Comment:
is this constructor still necessary? we might be able to redirect to the normal one these days

Reply:
Removed